presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -746,6 +746,113 @@ public void testCreateTableNonSupportedVarcharColumn()
         assertUpdate("CREATE TABLE test_create_table_non_supported_varchar_column (apple varchar(65536))");
     }
 
+    @Test
+    public void testEmptyBucketedTable()
+    {
+        // go through all storage formats to make sure the empty buckets are correctly created
```

Comment:
how does this relate to storage format?

Reply:
It shouldn't. However I don't want to rely on this. Also this is how we test other things like that in this class.